src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/4.diff.sql

```diff
@@ -0,0 +1,168 @@
+
+/*************************************************************
+Wrapping up in a transaction except CREATE FULLTEXT INDEX which is non-transactional script. Since there are no slow scripts(all the statements i.e. CREATE TABLE/INDEX/STORED PROC and ALTER STORED PROC are faster) so keeping all in one transaction.
+Guidelines to create migration scripts - https://github.com/microsoft/healthcare-shared-components/tree/master/src/Microsoft.Health.SqlServer/SqlSchemaScriptsGuidelines.md
+**************************************************************/
+SET XACT_ABORT ON
+BEGIN TRANSACTION
+
+/*************************************************************
+    Reindex State Table. 
+    Reindex state on each extended query tag.
+**************************************************************/
+IF NOT EXISTS (
+    SELECT * 
+    FROM sys.tables
+    WHERE name = 'ReindexState')
+BEGIN
+    CREATE TABLE dbo.ReindexState (
```

Comment:
While discussing with Will, we decided we do not need this. All we need is Tag table index status.

Reply:
Think like a framework who's concern is to handle the indexing internal state. Externaly all you care is passed or failed. To use it with QIDO.

Job framework also provides job status and error messages if failed

Reply:
We should just add Operation ID To existing tag table

Reply:
Thanks Smitha!
Discussed with Will. 
* How to store internal state is still in discussion.
* This PR would start with a simple solution , which will be incrementally improved later